spec/support/seed.rb

```diff
@@ -0,0 +1,47 @@
+class Seed
+  def self.airports
+    airports = [
+      { name: "N'djili Airport", location: "Kinshasa, Congo" },
+      { name: "Leon M'ba International Airport", location: "Libraville, Gabon" },
+      { name: "Kigali International Airport", location: "Kigali, Rwanda" },
+      { name: "Julius Nyerere International Airport",
+        location: "Dar es Salam, Tanzania" },
+      { name: "Addis Ababa Bole International Airport",
+        location: "Addis Ababa, Ethiopia" }
+    ]
+    @airlines = ["Dana Air", "Sosoliso", "Bell View", "Kenya Airways", "Arik Air",
+                "Virgin Atlantic", "Sname Air"]
+    airports.each do |airport|
+      Airport.create(airport)
+    end
+  end
+
+  def self.flights
+    10.times do
+      id = [1,2,3].sample
+      arr_id = [4,5].sample
+      flight = Flight.new
+      flight.duration = 12
+      flight.airline = @airlines.sample
+      flight.departure_id = id
+      flight.arrival_id  = arr_id
```

Comment:
Unnecessary spacing detected.<br>Operator `=` should be surrounded by a single space.